src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -454,6 +454,7 @@ private void storeResource(WsmResource resource) {
             .addValue("attributes", resource.attributesToJson());
     if (resource.getStewardshipType().equals(CONTROLLED)) {
       ControlledResource controlledResource = resource.castToControlledResource();
+      //noinspection deprecation
```

Comment:
What does that mean?

Reply:
It shushes the deprecation warning. My thinking ATM is that I want to warn on new uses and grandfather in the two existing ones.